CONTRIBUTING

```diff
@@ -0,0 +1,27 @@
+CONTRIBUTIONS and CODE REVIEWS
+
+We love contributions from the community. Please send pull requests and raise
+issues! All code submissions require review. We use GitHub pull requests for
+this purpose.
+
+
+COPYRIGHTS and LICENSE
+
+Except as otherwise noted in individual files, nMigen is licensed under the
```

Comment:
I don't believe we currently have any such notices, and I would *much* prefer to not have them in the future, so let's leave that part out. nMigen isn't a massive amalgamation that has to contend with things like large differently licensed vendored dependencies; it's a fairly small library.